en/advanced_config/bootloader_update.md

```diff
@@ -96,18 +97,25 @@ To update the bootloader:
    In this case you will not be able to install FMUv3 hardware.
    :::
 
-## Dronecode Probe Bootloader Update
+## Debug Probe Bootloader Update
 
-The following steps explain how you can "manually" update the bootloader using the dronecode probe:
+The following steps explain how you can "manually" update the bootloader using a [compatible Debug Probe](../debug/swd_debug.md#debug-probes-for-px4-hardware):
 
 1. Get a binary containing the bootloader (either from dev team or build it yourself).
-1. Connect the Dronecode probe to your PC via USB.
+
+1. Get a [Debug Probe](../debug/swd_debug.md#debug-probes-for-px4-hardware).
+   Connect the probe your PC via USB and setup the `gdbserver`.
+
 1. Go into the directory containing the binary and run the following command in the terminal:
 
    ```sh
    arm-none-eabi-gdb px4fmuv5_bl.elf
```

Comment:
```suggestion
   arm-none-eabi-gdb px4fmuv6x_bl.elf
```

Reply:
This is incorrect so legacy for V5 is 
`px4fmuv5_bl.elf`
Older version I don't even know.
For V6X it's
`px4_fmu-v6x_bootloader.elf`
For V6X-RT it's
`px4_fmu-v6xrt_bootloader.elf`

Reply:
In-tree H7 Booloaders from https://github.com/PX4/PX4-Autopilot are named *._bootloader all the others are from https://github.com/PX4/PX4-Bootloader and end with _bl. 

Using up a term like legacy will not help anyone nor will it be valid on the next bootloaderr.

Reply:
Thanks both. Fixed by listing the ones above and adding a note about the pattern.

> Using up a term like legacy will not help anyone nor will it be valid on the next bootloaderr.

You could not be more right. Sorry about that.